proto/slinky/mm2/v1/query.proto

```diff
@@ -0,0 +1,85 @@
+syntax = "proto3";
+package slinky.mm2.v1;
+
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+import "slinky/types/v1/currency_pair.proto";
+import "slinky/mm2/v1/market.proto";
+import "slinky/mm2/v1/params.proto";
+
+option go_package = "github.com/skip-mev/slinky/x/mm2/types";
+
+// Query is the query service for the x/marketmap module.
+service Query {
+  // MarketMap returns the full market map stored in the x/marketmap
+  // module.
+  rpc MarketMap(MarketMapRequest) returns (MarketMapResponse) {
+    option (google.api.http).get = "/slinky/mm2/v1/marketmap";
+  }
+
+  // Market returns a market stored in the x/marketmap
+  // module.
+  rpc Market(MarketRequest) returns (MarketResponse) {
+    option (google.api.http).get = "/slinky/mm2/v1/market";
+  }
+
+  // LastUpdated returns the last height the market map was updated at.
+  rpc LastUpdated(LastUpdatedRequest) returns (LastUpdatedResponse) {
+    option (google.api.http).get = "/slinky/mm2/v1/last_updated";
+  }
+
+  // Params returns the current x/marketmap module parameters.
+  rpc Params(ParamsRequest) returns (ParamsResponse) {
+    option (google.api.http) = {
+      get : "/slinky/mm2/v1/params"
+    };
+  }
```

Comment:
nit: what is mm2?

Reply:
This is the "temp" name we are naming this module. We already had a v0 version of the module named x/marketmap.

The plan is to replace that module with this one we are developing now. So the mm2 is just a temp name until we do the full switch over

Reply:
If I understand correctly, before this code is shipped to imported into our code base, you will be replacing `mm2`?